test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,51 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.IntStream;
+
+public final class ClientServiceFixtures {
+  public static final Long FAKEID = Long.valueOf(10);
+
+  public static List<Client> buildClients(Integer amount) {
+    List<Client> clients = new ArrayList<>();
+    IntStream.range(0,amount).forEach(i->clients.add(buildClient()));
```

Comment:
Use `map`

Reply:
i did this:
clients=IntStream.range(0,size).mapToObj(i->ClientServiceFixtures.buildClient()).collect(Collectors.toList());